core/src/block.rs

```diff
@@ -37,13 +35,12 @@ use crate::{
     tx::{TransactionValidator, VersionedAcceptedTransaction},
 };
 
-const PIPELINE_TIME_MS: u64 =
-    DEFAULT_BLOCK_TIME_MS + DEFAULT_COMMIT_TIME_LIMIT_MS + DEFAULT_TX_RECEIPT_TIME_LIMIT_MS;
+const PIPELINE_TIME_MS: u64 = DEFAULT_BLOCK_TIME_MS + DEFAULT_COMMIT_TIME_LIMIT_MS;
 
 /// Default estimation of consensus duration
 #[allow(clippy::integer_division)]
 pub const DEFAULT_CONSENSUS_ESTIMATION_MS: u64 =
-    (DEFAULT_COMMIT_TIME_LIMIT_MS + PIPELINE_TIME_MS) / 2;
+    DEFAULT_BLOCK_TIME_MS + (DEFAULT_COMMIT_TIME_LIMIT_MS / 2);
```

Comment:
I tried many different values for this constant and was unable to decrease the error. You could probably bring it down since you understand time triggers better. When sumeragi is unloaded, the shortest possible block time is simply the block time. The longest allowed block time is *block time* + *commit time*. The average of these two is *block time* + *commit time* / *2*.

Reply:
Ok, got it